samples/drivers/lora/lorawan/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _hello_world:
```

Comment:
This is a copy of the existing hello_world example (found in samples/hello_world/README.rst).  If you're using this as a starting template, great.  But this PR can't be merged as it is, and will get doc build errors because a doc with the name ``_hello_world`` already exists.

Reply:
Thanks for the reply, but the PR isn’t meant for merging yet but rather for discussion of the concept

Reply:
OK.  Ping me when it's ready for a doc review.